services/api/db/userSerializers.py

```diff
@@ -4,7 +4,7 @@
 class ProfileSerializer(serializers.ModelSerializer):
     class Meta:
         model = Profile
-        fields = ['bio', "date_of_birth", "profile_image_url"]
+        fields = ['bio', "date_of_birth", "theme", "profile_image_url"]
```

Comment:
```suggestion
        fields = ['bio', "date_of_birth", "theme", "profile_image_url"]
```
should we use fields = "__all__" as all the field are being included form the Profile model?

Reply:
No, returning "__all__" will also include "user" field of profile table which is created by auth user model (one to one field), sterilizer will expect it to be passed with json data we send to update with, that's not desirable. Alternate solution will be using [ exclude = ('user',)] , i will do that later when fields will increase.